src/factories/BuilderUtils.js

```diff
@@ -73,6 +73,25 @@ module.exports = ['FormioUtils', function(FormioUtils) {
     });
   };
 
+  /**
+   * Iterate the given label to make it unique.
+   *
+   * @param {String} label
+   *   Modify the component label to be unique.
+   *
+   * @returns {String}
+   *   The new component label.
+   */
+  var iterateLabel = function(label) {
```

Comment:
What if you have two "First Name" fields, but one of them is in a "Customer" container. This should be acceptable. Will this instead force them to make it say "First Name 2"?

Reply:
Do labels have to be unique? I didn't think that was a constraint.

Reply:
My complaint exactly. They should not have to be unique.

Reply:
Actually yes, you are right!

Reply:
The only change is in ngFormio